node/sc/bridge_manager_test.go

```diff
@@ -1509,8 +1505,6 @@ func TestAnchoringBasic(t *testing.T) {
 	curBlk := sim.BlockChain().CurrentBlock()
 
 	// Generate anchoring tx again for only the curBlk.
-	sc.handler.txCount = startTxCount
-	sc.handler.txCountEnabledBlockNumber = curBlk.NumberU64()
 	sc.handler.blockAnchoringManager(curBlk)
```

Comment:
Since the logic has been modified so that txCount is available immediately, how about modifying the comment for blockAnchoringManager?